dgen_os/python/input_data_functions.py

```diff
@@ -334,12 +369,12 @@ def melt_year(parameter_name):
     
     Parameters
     ----------
-    parameter name : 'string'
+    parameter_name : `string`
         Name of the parameter value in dataframe. 
     
     Returns
     -------
-    function : 'function'
+    function : `function`
         Function that melts years and parameter value to row axis
```

Comment:
```suggestion
        DataFrame with years and parameter value melted to row axis
```
This better describes what is happening from the internal function